package.json

```diff
@@ -58,7 +71,11 @@
     "production": [
       ">0.2%",
       "not dead",
-      "not op_mini all"
+      "not op_mini all",
+      "not ie <=11",
+      "not android <10",
+      "not safari <14",
+      "not ios_saf <14"
```

Comment:
Can you also change the `development` ones to make them the same

Reply:
Sure. Done!